src/app/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,43 +1,48 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.actnow">
-   <application
-        android:label="ActNow"
-        android:icon="@mipmap/ic_launcher">
-        <meta-data android:name="com.google.android.geo.API_KEY"
-               android:value="AIzaSyAXfUIU0H0eBvOoX73rB1_HJKiuZvd_eoQ"/>
-        <activity
-            android:name=".MainActivity"
-            android:launchMode="singleTop"
-            android:theme="@style/LaunchTheme"
-            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
-            android:hardwareAccelerated="true"
-            android:windowSoftInputMode="adjustResize">
-            <!-- Specifies an Android theme to apply to this Activity as soon as
-                 the Android process has started. This theme is visible to the user
-                 while the Flutter UI initializes. After that, this theme continues
-                 to determine the Window background behind the Flutter UI. -->
-            <meta-data
-              android:name="io.flutter.embedding.android.NormalTheme"
-              android:resource="@style/NormalTheme"
-              />
-            <!-- Displays an Android View that continues showing the launch screen
-                 Drawable until Flutter paints its first frame, then this splash
-                 screen fades out. A splash screen is useful to avoid any visual
-                 gap between the end of Android's launch screen and the painting of
-                 Flutter's first frame. -->
-            <meta-data
-              android:name="io.flutter.embedding.android.SplashScreenDrawable"
-              android:resource="@drawable/launch_background"
-              />
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
-                <category android:name="android.intent.category.LAUNCHER"/>
-            </intent-filter>
-        </activity>
-        <!-- Don't delete the meta-data below.
-             This is used by the Flutter tool to generate GeneratedPluginRegistrant.java -->
-        <meta-data
-            android:name="flutterEmbedding"
-            android:value="2" />
-    </application>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.example.actnow">
+	<application android:label="ActNow" android:icon="@mipmap/ic_launcher">
+		<meta-data android:name="com.google.android.geo.API_KEY" android:value="AIzaSyAXfUIU0H0eBvOoX73rB1_HJKiuZvd_eoQ" />
```

Comment:
hmm this is our android geo api key, we definitely want to remove this from github